cluster/gce/gci/configure_helper_test.go

```diff
@@ -147,8 +151,12 @@ func (c *ManifestTestCase) mustLoadPodFromManifest() {
 	}
 }
 
-func (c *ManifestTestCase) tearDown() {
-	os.RemoveAll(c.kubeHome)
+func (c *ManifestTestCase) tearDown() (err error) {
```

Comment:
don't need named return value

Reply:
I don't find any use of this return value, not sure why we are adding it.

Reply:
ok, I am sorry about that, And I will close it.